### ISSUE TITLE
Asking for recommendation when running multiple condition samples

### ISSUE THREAD

**Issue by jayhanbest** [ETH-NEXUS/scAmpi_single_cell_RNA#19]
Hi, I am trying to run scAmpi with a triplicate sample of 9 mouse.
The samples mainly divided in 3 conditions: Control 1,2,3 and Disease model 1,2,3 and Disease+drug 1,2,3.
In this case, would you recommend mapping all 9 samples together, or would it be better to map the three conditions separately and running the pipeline three times?
Additionally, which step would be appropriate to run workflow/scripts/sample_integration.R ?

**Comment by aricht** [comment]
Hi,
you can run scAmpi with all 9 samples in one run, each sample will be analysed independent of the others, but you have to start the pipeline only once and all results will be in the same directories. With this you can compare the results of the replicates with each other.

The script "sample_integration.R" can be viewed as a template/example script. We created it as part of our own cohort analyses, but it is not part of the scAmpi workflow and we cannot provide support on the highly complex topic of sample integration, this is beyond the scope of the pipeline. The script integrates a set of samples (reading in RDS files, e.g. those produced after the step "remove_atypical_cells") and highlights a specific sample against the cohort.

Best,
Anne
